tests/helpers/reporter.py

```diff
@@ -0,0 +1,10 @@
+from kitovu.utils import AbstractReporter
+
+
+class TestReporter(AbstractReporter):
+    def __init__(self, raise_errors=False):
```

Comment:
Wouldn't it make more sense for `raise_errors` to default to `True`?

Reply:
Hmmm, I'd say in the tests I'd mostly want a reportert that just ignores anything

Reply:
Ignoring unexpected warnings in tests (rather than making them fail tests) sounds like the wrong thing to do, IMHO :wink:

Reply:
Makes sense, I've updated it to `True`.